### PR TITLE
MGDSTRM-8538: add configuration for multiple clusters/regions via par…

### PR DESCRIPTION
**Pull request by sjhiggs**
…amter templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
...
+
+   * KUBE_CONFIG in `kas-fleet-manager-secrets-template-params`
+   * SUPPORTED_CLOUD_PROVIDERS in `kas-fleet-manager-service-template-params`
+   * CLUSTER_LIST in `kas-fleet-manager-service-template-params`
+
+The following sections describe the contents of these parameters in detail.
+
+#### SUPPORTED_CLOUD_PROVIDERS
+
+The SUPPORTED_CLOUD_PROVIDERS paramter contains a list of supported cloud providers in a yaml format.  See [deploying-kas-fleet-manager-to-openshift.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md) for more details.
```
Comment (MikeEdgar) [inline README.md:188]:
```suggestion
The SUPPORTED_CLOUD_PROVIDERS parameter contains a list of supported cloud providers in a yaml format.  See [deploying-kas-fleet-manager-to-openshift.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md) for more details.
```

**Review by MikeEdgar — COMMENTED** [review]
@sjhiggs , this looks good.

**Comment by MikeEdgar** [comment]
@machi1990 - please take a look. This is for #158 .

File: README.md [hunk 2]
```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
 managed_kafka.sh --create mykafka --plan standard.x2
 ```
 
+### Deploying to multiple clusters/zones
+In order to allow clients to deploy Kafka clusters to multiple clusters and/or zones, the following parameters must be customized:
+
+   * KUBE_CONFIG in `kas-fleet-manager-secrets-template-params`
```
Comment (machi1990) [inline README.md:180]: This is needed to be customizable only when running in standalone mode.
When using OCM, this won't be needed.
/cc @MikeEdgar

File: README.md [hunk 3]
```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
...
+The SUPPORTED_CLOUD_PROVIDERS paramter contains a list of supported cloud providers in a yaml format.  See [deploying-kas-fleet-manager-to-openshift.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md) for more details.
+
+
+#### CLUSTER_LIST
+
+The CLUSTER_LIST parameter contains a list clusters for kas fleet manager.  See [data-plane-osd-cluster-options.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/data-plane-osd-cluster-options.md) for more details.
+
+#### KUBE_CONFIG
+
+The KUBE_CONFIG contains cluster connection information for each cluster to which Kafka can be deployed.  There should be one entry in KUBE_CONFIG for each entry in CLUSTER_LIST.  For example, to generate a KUBE_CONFIG for two clusters:
```
Comment (machi1990) [inline README.md:197]: Might be worth putting a note that this is only needed when running in standalone mode. 
When using OCM, this is not needed. Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/187/files#r899930511

File: README.md [hunk 4]
```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
...
+
+```shell
+oc login -u kubeadmin -p <first_cluster_password> <first_cluster_url>
+oc config view --minify --raw > /tmp/c1.yaml
+oc login -u kubeadmin -p <second_cluster_password> <second_cluster_url>
+oc config view --minify --raw > /tmp/c2.yaml
+(KUBECONFIG=/tmp/c1.yaml:/tmp/c2.yaml oc config view --raw)
+```
```
Comment (machi1990) [inline README.md:205]: nice trick!

**Comment by machi1990** [comment]
Looks good overall. I've left a suggestion on noting that kubeconfig only needs to be configured when running in standalone

**Comment by sjhiggs** [comment]
@MikeEdgar @machi1990 I updated the docs to reflect your suggestions.  I will note that I didn't test OCM mode installation.

**Comment by machi1990** [comment]
Thank you

**Pull request merged**